substrate/primitives/core/src/sr25519.rs

```diff
@@ -499,11 +488,13 @@ impl TraitPair for Pair {
 		Ok((Self(result.into()), seed.map(|s| MiniSecretKey::to_bytes(&s))))
 	}
 
+	#[cfg(feature = "full_crypto")]
 	fn sign(&self, message: &[u8]) -> Signature {
 		let context = signing_context(SIGNING_CTX);
 		self.0.sign(context.bytes(message)).into()
 	}
 
+	#[cfg(feature = "full_crypto")]
 	fn verify<M: AsRef<[u8]>>(sig: &Signature, message: M, pubkey: &Public) -> bool {
```

Comment:
Should also work on `no_std`?

Reply:
It should work. I am not sure if we should enabled it. We don't really need it at this point.

Reply:
1b57b65